### ISSUE TITLE
Question regarding data preparation

### ISSUE THREAD

**Issue by MCorentin** [david-hirst/MOTL#6]
Hi,

Thanks for MOTL.

I am trying to apply MOTL to my own dataset, using the TGCA learning dataset factorization files and I have a question regarding data preparation. 

In the Readme it is recommended to use raw counts for mRNA (https://github.com/david-hirst/MOTL?tab=readme-ov-file#pre-processing-of-target-dataset-factorization) however, in the paper it is stated that the training dataset was trained using log2(x+1) counts. And when applied to the gliobastoma dataset, the mRNA counts were also normalised and transformed using log(x + 1).

Could you confirm if I should use the raw counts or the normalised log2(x + 1) counts for the mRNA target dataset ?

Thanks,
Corentin

**Comment by david-hirst** [comment]
Hi Corentin,

Thanks for your interest in MOTL.

In the readme workflow you'll notice the following section of code:

`YTrg_list = TargetDataPreparation(views = views, YTrg_list = YTrg_list, 
                                  Fctrzn = Fctrzn, smpls = smpls, expdat_meta_Lrn = expdat_meta_Lrn,
                                  normalization = 'Lrn', transformation = TRUE)`

This function performs the normalisation and transformation of raw counts contained in `YTrg_list`. 
By setting  `normalization = 'Lrn'` the function normalises mRNA and miRNA counts with the DESeq2 method,  using the geo-means from the DESeq2 normalisation of the learning dataset. 
These geo-means are stored in the learning set metadata object. As mentioned in the readme, the metadata for the learning dataset is downloaded form zenodo and then imported into your R session with:

`expdat_meta_Lrn = readRDS(file.path(LrnDir,"expdat_meta.rds"))`

If you wish to do your own normalisation / transformation - you can do that before running `TargetDataPreparation(..., normalization = FALSE, transformation = FALSE)`

FYI: We are currently working on providing MOTL functionality as a bioconductor package with improved instructions.

Regards
David

**Comment by MCorentin** [comment]
Thank you for your answer, everything is clear now !

Regards,
Corentin
